mkosi.profiles/kiosk/mkosi.conf

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=fedora
+
+[Content]
+Packages=
+        @base-x
```

Comment:
What exactly does this pull in? A comment with a link to the package list would be great.

Reply:
It's this one: https://pagure.io/fedora-comps/blob/main/f/comps-f42.xml.in#_281

I suppose we could expand it (and drop plymouth and some of the weird drivers), or possibly fold it into the desktop profile as it's kinda a base requirement for getting graphics to work on Fedora.

Reply:
I would definitely expand it and get rid of all the xorg stuff. I think we're OK with only supporting wayland.

Reply:
And adding all of that to the desktop profile seems like a good fit

Reply:
Done